src/libraries/System.IO.FileSystem/tests/File/GetSetAttributes.cs

```diff
@@ -1,22 +1,40 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using Microsoft.Win32.SafeHandles;
 using Xunit;
 
 namespace System.IO.Tests
 {
     public class File_GetSetAttributes : BaseGetSetAttributes
     {
         protected override FileAttributes GetAttributes(string path) => File.GetAttributes(path);
+
+        protected FileAttributes GetAttributes(SafeFileHandle fileHandle) => File.GetAttributes(fileHandle);
```

Comment:
Same comment as https://github.com/dotnet/runtime/pull/60507/files#r807786215.
Maybe the SafeHandle API should be in its own class. That class would override the methods from `BaseGetSetAttributes`:
```cs
        protected abstract FileAttributes GetAttributes(string path);
        protected abstract void SetAttributes(string path, FileAttributes attributes);
```
and implement them using the `SafeFileHandle` API.